lib/components/form/user-settings.js

```diff
@@ -42,7 +42,26 @@ function findLocationType(
     ? getFormattedPlaces(match.type, intl)
     : null
 }
-function summarizeQuery(query, intl, locations = []) {
+export function summarizeQuery(query, intl, locations = []) {
+  if (!query.from.name) {
+    query.from.name = intl.formatMessage(
```

Comment:
In English at least, he code on line 67 interferes with this logic (only the latitude will be shown in the title bar as a result). But also, for conciseness it is nice to continue stripping the city, state, and zip of actual addresses.

Reply:
I've added a new method to only do the split if any alphabetical character is found! Let me know what you think...